_component_design/selector-strip.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Selector Strip
+parent: Inputs and Controls
+layout: component
```

Comment:
Shouldn't this have a `secondary` metadata value of `Selectors`?

Reply:
Added, will function once pull/323 is merged.